src/app/(proper_react)/(redesign)/(public)/how-it-works/components/SectionTwo.tsx

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import { signIn } from "next-auth/react";
+import { Props } from "../HowItWorksView";
+import { TelemetryButton } from "../../../../../components/client/TelemetryButton";
+import { Resolve } from "../images";
+import styles from "../HowItWorksView.module.scss";
+
+export const SectionTwo = (props: Props) => {
+  const { l10n } = props;
+
+  return (
+    <div id="data-breach-section" className={styles.sectionWrapper}>
+      <div className={styles.intro}>
+        <span className={styles.introContent}>
+          <h2>
+            {l10n.getFragment("section-2-intro-title", {
+              elems: {
+                data_breaches: <span className={styles.breachesEmphasis} />,
+              },
+            })}
+          </h2>
+          <p>{l10n.getString("section-2-intro-text-1")}</p>
+          <hr className={styles.horizontalRule} />
+        </span>
+        <div className={styles.introCTA}>
+          <h3>{l10n.getString("section-2-intro-text-2")}</h3>
+          <div className={styles.sectionCTAButton}>
+            <TelemetryButton
+              variant="primary"
+              onPress={() => {
+                void signIn("fxa");
+              }}
+              event={{
+                module: "ctaButton",
+                name: "click",
+                data: {
+                  button_id: "free_scan_first",
+                },
+              }}
+            >
+              {l10n.getString("section-2-intro-cta-button")}
+            </TelemetryButton>
+          </div>
+        </div>
+      </div>
+      <div className={`${styles.step} ${styles.mobileColumns}`}>
+        <div className={`${styles.stepTextContainer} ${styles.mobileColumns}`}>
```

Comment:
Suggestion: Semantically, it would make sense for this content to be structured using a list — `ul` or even `ol` … or was there a specific reason for not using a list?

Reply:
Update here:
https://github.com/mozilla/blurts-server/pull/4632/commits/5bb3f7bcbb81b91115cfa736aee896313745cd82